### ISSUE TITLE
keyboard accessibility of "Trusted by" widget on the Gatsby homepage

### ISSUE THREAD

**Issue by WilcoFiers** [gatsbyjs/gatsby#22214]
I noticed this issue when running [axe](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd) on the www.gatsbyjs.org. The "Trusted by" widget shows a series of logos that can be scrolled horizontally. Because the scrollable element is not focusable, and there is no focusable element inside the `div` there is no way for a keyboard user to scroll this element. It's a fairly minor issue, but I imagine keyboard users would also want to be able to see all the logos in this list. 

The fix for this is pretty straight-forward; Put `tabindex=0` on the `div` with `overflow-x: scroll` on it.

![Screenshot 2020-03-12 11 05 10](https://user-images.githubusercontent.com/530687/76541535-0a549f00-6484-11ea-9fc9-1f124a1ee71e.png)

**Comment by pieh** [comment]
Thanks for report and suggestion how to fix it!

Drilling down code a bit - seems like `<LogoGrup>` does need `tabindex` attribute ( https://github.com/gatsbyjs/gatsby/blob/eb94fe9d4bbe4a51e03127d3596a92838fba8176/www/src/components/homepage/homepage-logo-banner.js#L93 )

**Comment by marcysutton** [comment]
@WilcoFiers, thanks for reporting! Question for you: does the `tabindex` on a DIV without an interactive role provide an issue for assistive technology users, who won't have that signal? Or even issues with interaction modes in NVDA/JAWS? I can test it later if someone doesn't beat me to it. cc @madalynrose.

**Comment by marcysutton** [comment]
Following up on this, I created a Codepen demo to test it: https://codepen.io/marcysutton/pen/xxGJgNy

In Chrome and Voiceover, it works fine from a keyboard mechanics perspective. It will also read out the alt text for images by default, but this isn't necessarily clear to AT users.

In Safari and Voiceover, additional keys are needed to scroll: I was able to navigate through the content using Shift and arrow keys, but it actually put the virtual cursor on each image and line break in my demo. This seems fine if users know how to use Voiceover, but the lack of a label for why this region is focusable still seems like an issue. It announces the alt text for images by default.

In NVDA and Firefox, it announces the alt text for the images by default as well. But to scroll the region, I have to hold the Insert key to pass the arrow key command through to the browser. Without a label for the wrapping element, this could be confusing.

So I believe my original hunch was accurate, that we likely need to label these focusable regions and potentially add an interactive role for the key mapping issues-it's unclear to me which role to pick, however. Thought you should know @WilcoFiers since it applies to your/axe's advice.

**Comment by marcysutton** [comment]
Closed with #23720.
